schema/mutation/editForm.ts

```diff
@@ -144,6 +144,9 @@ export default {
         }
         if (args.name) {
             update.name = args.name;
+            await Resource.findByIdAndUpdate(form.resource, {
+                name: args.name
```

Comment:
@adrianmujica153 did that work only if form is the core of the resource ?

Reply:
Yes, because only the forms that are "core" has the property "resource"

Reply:
I think inherited forms also, I'll check

Reply:
you are right